e2e_tests/resources/deployment.py

```diff
@@ -32,9 +32,20 @@ async def check_deployment(client, operation_endpoint, headers):
     response = await client.get(full_endpoint, headers=headers, timeout=TIMEOUT)
     if response.status_code == 200:
         deployment_status = response.json()["operation"]["status"]
```

Comment:
Now there're 2 response.parse, maybe you should consolidate?

Reply:
done